### PR TITLE
Collect openQA logs in all tests

### PR DESCRIPTION
**Pull request by okurz**
Verification run: http://lord.arch/tests/5540

### CODE REVIEW

**Comment by okurz** [comment]
ok, that did not actually work but I can see that the problem is OOM so let's add a bit of RAM.

**Comment by okurz** [comment]
fixed

File: lib/openQAcoretest.pm [hunk 1]
```diff
@@ -1,8 +1,24 @@
 package openQAcoretest;
 use base "basetest";
+use testapi;
 
-# All steps belonging to core openQA functionality are 'fatal'. by default
 
+sub get_log {
+    my ($cmd, $name) = @_;
+    my $ret = script_run "$cmd | tee $name";
+    upload_logs $name if $ret;
```
Comment (coolo) [inline lib/openQAcoretest.pm]: as you're asking for my review: I suggest to use () if you use post-if

Comment (okurz) [inline lib/openQAcoretest.pm]: sure, feedback appreciated, done

**Comment by okurz** [comment]
no, wait. failed

**Comment by okurz** [comment]
fixed

**Pull request merged**